ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -260,7 +260,7 @@ mkNodeArgs registry cfg initState tracers btime chainDB = NodeArgs
     , blockFetchSize      = nodeBlockFetchSize
     , blockMatchesHeader  = nodeBlockMatchesHeader
     , maxUnackTxs         = 100 -- TODO
-    , mempoolCap          = MempoolCapacity 1000
+    , mempoolCap          = MempoolCapacity 1000000 -- TODO
```

Comment:
@mrBliss: Eventually this should be configurable, but any idea what a good `MempoolCapacity` would be?

Reply:
@dcoutts: can you suggest a good default capacity in bytes for the Mempool of the node?

Reply:
A sensible default is 2x the current max block size.

Ultimately we should allow a local override from the node config file. The benchmark team will want this.

Reply:
For the time being, I've hardcoded this to `128_000` bytes (2x the mainnet max block size).

Reply:
Oops, I got this wrong. `128_000` is 2x the mainnet max **transaction** size. From the [mainnet config](https://github.com/input-output-hk/cardano-node/blob/master/configuration/mainnet-ci/configuration.yaml#L292) in `cardano-node`, the max block size appears to be `2_000_000`.